package.json

```diff
@@ -29,7 +29,7 @@
     "pg:build": "pgtyped -c ./packages/server/postgres/pgtypedConfig.js",
     "pg:generate": "export $(grep ^POSTGRES_ .env | tr -d \"'\"); yarn kysely-codegen --exclude-pattern \"(PgMigrations|StripeQuantityMismatchLogging)\" --out-file ./packages/server/postgres/types/pg.d.ts --dialect postgres --url postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB && prettier --write ./packages/server/postgres/types/pg.d.ts",
     "pg:restore": "node ./scripts/toolbox/pgRestore.js",
-    "db:start": "docker compose -f docker/stacks/development/docker-compose.yml up -d",
+    "db:start": "mkdir -p dev && docker compose -f docker/stacks/development/docker-compose.yml up -d",
```

Comment:
+1 I'm ok with this, but I think your docker might be misconfigured. Mine creates the folder with me as owner.

Reply:
I think it's just because by default Docker installs with root privileges in Linux. Docker daemon runs as root and you have to add your user to a group called `docker` so you can use docker commands without doing `sudo docker XXX`. That's what I did long time ago and it's still what's explained in [here](https://docs.docker.com/engine/install/linux-postinstall/).

Apparently they might have added some documentation around [running Docker in root-less mode](https://docs.docker.com/engine/security/rootless/), but it is clearly not the default behavior. I think we should keep this here for everyone using Linux and installing Docker in its default way.

Reply:
Can we just get this part of the change in?

Reply:
Yes, ofc. I corrected the branch so the only change is this one.